lira/src/bayes-image-analysis.c

```diff
@@ -1643,8 +1657,8 @@ double lpost_alpha(double alpha, /* evaluate at alpha */
   } /* loop over rows of ms.ag[level] */
 
   /****************  ADD THE LOG PRIOR *************/
-  logpost -= (   ms->al_kap1 *log(alpha) 
-	       + ms->al_kap2 * pow( alpha, ms->al_kap3 ) );
+    logpost -= ( -ms->al_kap1 * log(ms->al_kap2)  +  ms->al_kap1 * log(alpha) 
+	        + ms->al_kap2 * pow( alpha, ms->al_kap3 ) );
```

Comment:
@infinitron I am wondering if requiring `al_kap1 < 0` would be enough to deal with the sign?